tdrs-frontend/src/components/EditProfile/EditProfile.test.js

```diff
@@ -493,4 +501,115 @@ describe('EditProfile', () => {
 
     expect(select.instance().value).toEqual('')
   })
+
+  it('routes "/edit-profile" to the Unassigned page when user has requested access', () => {
```

Comment:
Added test to make sure that the user is rerouted to `/request` if they have requested access.